scss/bootstrap/overrides/style/_button-groups.scss

```diff
@@ -1,27 +1,16 @@
-.btn .caret {
-  margin: 0 5px;
-}
-
 .btn-group {
-
-  .btn {
-    box-shadow: none;
+  label,
+  label:hover {
     border-color: $input-border;
+  }
 
-    &:not([class*=" btn-"]) {
-      &:hover {
-        border-color: $input-border;
-      }
-    }
-
-    // Adds missing border color on button hover:
-    &:hover {
-      border-color: $crm-grayblue-darker;
-    }
+  .active {
+    background-color: $gray;
+    border-color: transparent;
+    color: $crm-white;
+  }
 
-    &.active {
-      background-color: $gray;
-      color: $crm-white;
-    }
+  button:not([class*='-outline']) {
```

Comment:
Is this style necessary? I see it's being overridden by
```scss
#bootstrap-theme .btn.active {
  border-color: transparent;
}
```

Reply:
Yes, removed.